src/components/MessagesList/MessagesList.vue

```diff
@@ -931,10 +933,10 @@ export default {
 				this.displayMessagesLoader = false
 				this.debounceUpdateReadMarkerPosition()
 				return
+			} else if (scrollHeight > clientHeight) {
+				this.setChatScrolledToBottom(false)
```

Comment:
After swtiching to conversation, handleScroll is triggered. Regardless of container height, it sets value to false (whereas it shouldn't for non-scrollable chats)